scripts/ci/jobs/release-mgmt.sh

```diff
@@ -30,10 +30,6 @@ release_mgmt() {
 
     fi
 
-    "$ROOT/release/scripts/scan-images-with-roxctl.sh" || {
```

Comment:
@gavin-stackrox Does the `release-mgmt.sh` script run for X.Y.Z tags and release candidates? 
If so, because the GHA workflow does not build these tags yet, we would not scan the images any more IIUC.

Reply:
@tommartensen _currently_ that depends on the config in the openshift/releases repo and the versions in the release branches. So existing release patches and RCs will still be scanned this way. The next release should use GHA but how that will function is TBD.

Reply:
Ok, just to make it absolutely clear to me: we remove the image scan from this script here, and this script is used by openshift/releases, and we do not migrate to GHA for the release building in time, do we scan the images or not?
Thanks :)

Reply:
We *have* to migrate image builds to GHA for the next release so we will scan images via GHA on the next release branches. (I say we *have* to because it is not really feasible or beneficial to support two varieties of build systems)

Reply:
Understood.